pandas-stubs/core/series.pyi

```diff
@@ -208,55 +209,52 @@ class Series(IndexOpsMixin, NDFrame, Generic[S1]):
     _ListLike: TypeAlias = ArrayLike | dict[_str, np.ndarray] | list | tuple | Index
     __hash__: ClassVar[None]
 
+    # TODO: can __new__ be converted to __init__? Pandas implements __init__
     @overload
     def __new__(
         cls,
         data: DatetimeIndex | Sequence[Timestamp | np.datetime64 | datetime],
         index: Axes | None = ...,
-        dtype=...,
+        dtype: Dtype = ...,
         name: Hashable | None = ...,
         copy: bool = ...,
-        fastpath: bool = ...,
     ) -> TimestampSeries: ...
     @overload
-    def __new__(
+    def __new__(  # type: ignore[misc]
         cls,
         data: _ListLike,
-        dtype: Literal["datetime64[ns]"],
         index: Axes | None = ...,
+        *,
+        dtype: Literal["datetime64[ns]"],
```

Comment:
Probably should be `TimestampDtypeArg`

Reply:
I think all the other overloads would also need that: one overload with a specific `data` and one for `_ListLike` but a specific `dtype`?

Reply:
My concern there would be that it would be too wide.  In other words, there would be contents of lists that wouldn't work with certain dtypes.  I think the one is in there for `TimestampSeries` because it handles any kind of list.

Reply:
> Probably should be `TimestampDtypeArg`

I think this is the only change left to make.

Reply:
changed